src/commands/githubActions/endToEndTest/testCheckers.ts

```diff
@@ -116,10 +116,42 @@ async function ensureNoErrorLogs(
 }
 
 /**
- * Test that the validators are attesting after a peri
+ * Test that the validators are attesting after 16 minutes
+ * 
  */
-async function attestanceProof(network: Network): Promise<void> {
-  if (network === "mainnet") return;
-  // TODO
+export async function attestanceProof(network: Network): Promise<void> {
+  // Check if the network is mainnet or prater
+  if (network === "mainnet") {
+    // TODO
+    return;
+  } else if (network === "prater") {
+    let iterationCount = 1;
+    console.log(chalk.grey(`  - Checking if validator is active`));
+
+    while (iterationCount <= 9) {
+      await new Promise(resolve => setTimeout(resolve, 2 * 60 * 1000)); // Wait for 2 minutes before each iteration
+      const response = await got(`https://prater.beaconcha.in/api/v1/validator/${process.env.VALIDATOR_INDEX}`, {
```

Comment:
Also, here we are depending on beaconcha.in. Maybe we can use the Eth node we maintain to perform this check. Moreover, (at least on their UI) beaconcha.in has some minutes delay to show attestations, so this could be a problem here